bobtemplates/plone/view/browser/views.py.example.bob

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from Products.Five.browser import BrowserView
+# -*- Extra imports go here -*-
+
+
```

Comment:
Add basic BrowserView class here.

Reply:
I've already added this in a different branch in my fork. I'll merge it with my main branch when I'm done with the rest of the requested changes